samples/attested_tls/server/enc/Makefile

```diff
@@ -22,7 +23,7 @@ build:
 		--search-path $(INCDIR)/openenclave/edl/sgx
 	$(CXX) -g -c $(CXXFLAGS) $(INCLUDES) -std=c++11 -DSERVER_CERT_VERIFY_CALLBACK ecalls.cpp server.cpp ../../common/identity_verifier.cpp ../../common/cert_verifier.cpp ../../common/utility.cpp
 	$(CC) -c $(CFLAGS) $(CINCLUDES) tls_server_t.c
-	$(CXX) -o tls_server_enc ecalls.o server.o cert_verifier.o identity_verifier.o utility.o tls_server_t.o $(LDFLAGS) -lmbedtls -lmbedcrypto -loehostsock -loehostresolver -loecore -loelibc -loesyscall
+	$(CXX) -o tls_server_enc ecalls.o server.o cert_verifier.o identity_verifier.o utility.o tls_server_t.o $(LDFLAGS) $(CRYPTO_LDFLAGS) -loehostsock -loehostresolver
```

Comment:
Why have oecore, oelibc and oesyscall libraries been removed from the linker line?

Reply:
As `$(CRYPTO_LDFLAGS)` include them, they seem redundant.

Reply:
This changes the order of libraries which may break if there is an inherent ordering dependency among the libraries.

Reply:
I built and ran sample application after removing those flags. Found no issues.